src/pages/index.js

```diff
@@ -0,0 +1,22 @@
+import Link from "next/link";
+import Header from "../components/Header";
+import PageTitle from "../components/PageTitle";
+
+export default function Home() {
+  return (
+    <div>
+      <PageTitle title='Home' />
+      <Header />
+      <main>
+        <h1>Accessibility Cheatsheets for Components
+        </h1>
+        <ul>
+          <li><Link href='/accordion'><a>Accordion</a></Link></li>
+          <li><Link href='/dialog'><a>Dialog</a></Link></li>
+          <li><Link href='/disclosure'><a>Disclosure</a></Link></li>
+          <li><Link href='/tabs'><a>Tabs</a></Link></li>
+        </ul>
+      </main>
+    </div>
+  );
+}
```

Comment:
Instead of modifying `index.jsx`, this `index.js` file was created and is being ignored by Next. I would copy the contents of this into `index.jsx` and then delete this file.